### DISCUSSION TITLE
Solr push params

### DISCUSSION THREAD

**Discussion by Git-Romer** [irgroup-classrooms/dis17-2021#21] [Q&A] [answered]
Hi @phschaer 
I just looked at the indexing example and wondered what exactly the parameters do or if they are documented somewhere. Unfortunately I couldn't find the information I was looking for. Could you please provide the link to it?

**Comment by phschaer** [comment, marked as answer]
The documentation for the CSV import can be found here: https://solr.apache.org/guide/8_10/post-tool.html#indexing-csv

And here: https://solr.apache.org/guide/8_10/uploading-data-with-index-handlers.html#csv-formatted-index-updates

HTH

**Reply by Git-Romer** [reply]
Perfect!
Thank you very much
